scripts/release.sh

```diff
@@ -422,10 +432,13 @@ function parse_flags() {
     (( has_gcr_flag )) && echo "Not publishing the release, GCR flag is ignored"
     (( has_gcs_flag )) && echo "Not publishing the release, GCS flag is ignored"
     KO_DOCKER_REPO="ko.local"
-    KO_FLAGS="-L ${KO_FLAGS}"
     RELEASE_GCS_BUCKET=""
+    [[ -z "${RELEASE_DIR}" ]] && RELEASE_DIR="${REPO_ROOT_DIR}"
   fi
 
+  [[ -z "${RELEASE_GCS_BUCKET}" && -z "${RELEASE_DIR}" ]] && abort "--release-gcs or --release-dir must be used"
+  [[ -n "${RELEASE_DIR}" ]] && mkdir -p "${RELEASE_DIR}" 2> /dev/null
```

Comment:
I think it might be better to remove the `2> /dev/null` since if the dir already exists then `mkdir` remains silent due to the `-p`, so the only time we should see stderr text is when something is really bad, like we don't have permission to create the dir - which should be shown to the user (and probably stop processing)

Reply:
Good catch. Fixed.